[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.address`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.address` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.automotive`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.automotive` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.bank`.
[35mDEBUG   [0m faker.factory:factory.py:101 Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.barcode`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.barcode` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.color`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.color` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.company`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.company` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.credit_card`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.credit_card` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.currency`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.currency` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.date_time`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.date_time` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:113 Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.geo`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.geo` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.internet`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.internet` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:113 Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.job`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.job` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.lorem`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.lorem` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.misc`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.misc` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.person`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.person` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.phone_number`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.phone_number` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:113 Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.
[35mDEBUG   [0m faker.factory:factory.py:113 Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.
[35mDEBUG   [0m faker.factory:factory.py:94 Looking for locale `en_US` in provider `faker.providers.ssn`.
[35mDEBUG   [0m faker.factory:factory.py:106 Provider `faker.providers.ssn` has been localized to `en_US`.
[35mDEBUG   [0m faker.factory:factory.py:113 Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'get', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/gettoken', 'params': {'corpid': 'ww876064acebf0fa3c', 'corpsecret': 'A7LgEhs_Ty_dYXO9BcgY04PJBdawQ6JPzxNQJpv9YxY'}}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "GET /cgi-bin/gettoken?corpid=ww876064acebf0fa3c&corpsecret=A7LgEhs_Ty_dYXO9BcgY04PJBdawQ6JPzxNQJpv9YxY HTTP/1.1" 200 277
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'ok', 'access_token': 'amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g', 'expires_in': 7200}
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'get', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "GET /cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g HTTP/1.1" 200 100
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'ok', 'department': [{'id': 1, 'name': 'Hogwarts', 'parentid': 0, 'order': 100000000}]}
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'post', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/create?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g', 'json': {'name': 'ÂπøÂ∑ûÁ†îÂèë‰∏≠ÂøÉ2', 'name_en': 'RDGZ', 'parentid': 1, 'order': 1, 'id': 4}}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "POST /cgi-bin/department/create?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g HTTP/1.1" 200 39
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'created', 'id': 4}
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'get', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "GET /cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g HTTP/1.1" 200 178
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'ok', 'department': [{'id': 1, 'name': 'Hogwarts', 'parentid': 0, 'order': 100000000}, {'id': 4, 'name': 'ÂπøÂ∑ûÁ†îÂèë‰∏≠ÂøÉ2', 'parentid': 1, 'order': 1, 'name_en': 'RDGZ'}]}
[32mINFO    [0m root:base_api.py:22 ['Hogwarts', 'ÂπøÂ∑ûÁ†îÂèë‰∏≠ÂøÉ2']
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'post', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/update?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g', 'json': {'id': 4, 'name': 'ÂπøÂ∑ûÁ†îÂèë‰∏≠ÂøÉ-update', 'name_en': 'RDGZ', 'parentid': 1, 'order': 1}}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "POST /cgi-bin/department/update?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g HTTP/1.1" 200 32
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'updated'}
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'get', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "GET /cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g HTTP/1.1" 200 184
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'ok', 'department': [{'id': 1, 'name': 'Hogwarts', 'parentid': 0, 'order': 100000000}, {'id': 4, 'name': 'ÂπøÂ∑ûÁ†îÂèë‰∏≠ÂøÉ-update', 'parentid': 1, 'order': 1, 'name_en': 'RDGZ'}]}
[32mINFO    [0m root:base_api.py:22 ['Hogwarts', 'ÂπøÂ∑ûÁ†îÂèë‰∏≠ÂøÉ-update']
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'get', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/delete?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g&id=4'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "GET /cgi-bin/department/delete?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g&id=4 HTTP/1.1" 200 32
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'deleted'}
[32mINFO    [0m root:base_api.py:22 -----request data-----
[32mINFO    [0m root:base_api.py:22 {'method': 'get', 'url': 'https://qyapi.weixin.qq.com/cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g'}
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:939 Starting new HTTPS connection (1): qyapi.weixin.qq.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:433 https://qyapi.weixin.qq.com:443 "GET /cgi-bin/department/list?access_token=amRqN0wrjjj0ynHbP1rF0QvHUosJhKNzQO7t0AZ-Uz8oKsMXmnd7Rt-I0jQl15F0JtMyqc3Hp953-s0fglAIFY8eQRwi71JEMxR0-Ci-eeYdhtI3DeTHvFzaCnqqA3Sdd6x-Kn13l57z9uWJi8sw4iwEINMw1QVyuzCf23tBxIrrPIUR1DWBXLp4PCsrFuBaLOS48slaRsx4439zASEK6g HTTP/1.1" 200 100
[32mINFO    [0m root:base_api.py:22 -----response data-----
[32mINFO    [0m root:base_api.py:22 {'errcode': 0, 'errmsg': 'ok', 'department': [{'id': 1, 'name': 'Hogwarts', 'parentid': 0, 'order': 100000000}]}
[32mINFO    [0m root:base_api.py:22 [1]